### ISSUE TITLE
NameError: global name 'new_path_from_node' is not defined

### ISSUE THREAD

**Issue by bzamecnik** [martymcguire/inkscape-openscad-poly#3]
I opened an SVG file and tried to export it to OpenSCAD using *File -> Save a copy*. It crashed with the following message:

```
Traceback (most recent call last):
  File "openscad-poly.py", line 45, in <module>
    e.affect()
  File "/Applications/Inkscape.app/Contents/Resources/share/inkscape/extensions/inkex.py", line 268, in affect
    self.effect()
  File "openscad-poly.py", line 39, in effect
    parser.parse()
  File "/Users/bzamecnik/.config/inkscape/extensions/openscad_poly/svg_parser.py", line 231, in parse
    self.recursivelyTraverseSvg(self.svg, [[0.28222, 0.0, -(self.svgWidth/2.0)], [0.0, -0.28222, (self.svgHeight/2.0)]])
  File "/Users/bzamecnik/.config/inkscape/extensions/openscad_poly/svg_parser.py", line 288, in recursivelyTraverseSvg
    entity = self.make_entity(node, matNew)
  File "/Users/bzamecnik/.config/inkscape/extensions/openscad_poly/svg_parser.py", line 302, in make_entity
    entity.load(node,mat)
  File "/Users/bzamecnik/.config/inkscape/extensions/openscad_poly/svg_parser.py", line 135, in load
    newpath = new_path_from_node(node)
NameError: global name 'new_path_from_node' is not defined
```

This is the SVG:
```
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 17.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="5096.659px" height="1203.185px" viewBox="0 0 5096.659 1203.185" enable-background="new 0 0 5096.659 1203.185"
	 xml:space="preserve">
<path d="M3292.065,752.765l-61.176-173.236c-6.399,90.18-23.767,160.539-52.24,213.287L3292.065,752.765z"/>
<path d="M2539.363,171.933l-275.14,97.162c114.564,13.474,193.766,90.249,193.766,221.161v47.074
	c0,88.113-27.763,149.672-101.39,176.226c76.042,24.141,101.39,94.147,101.39,185.882v35.003c0,48.227-9.746,89.341-28.094,122.789
	l388.361-137.144h-278.892V171.933z"/>
<path d="M3006.162,494.167c0-213.828-32.84-240.999-87.965-240.999h-151.298v473.173h151.298
	C2985.05,726.341,3006.162,660.74,3006.162,494.167z"/>
<path d="M2223.826,892.195v-10.863c0-55.524-33.796-57.937-111.046-61.559l-196.745-8.448v-190.71l196.745-12.07
	c83.285-3.622,113.46-3.622,113.46-59.145v-10.863c0-61.559-50.696-61.559-113.46-61.559h-278.822v-55.941l-83.205,29.383
	l176.467,499.712h178.318C2199.685,950.132,2223.826,948.926,2223.826,892.195z"/>
<polygon points="2016.582,1203.185 2135.017,1161.361 2001.813,1161.361 "/>
<path d="M3053.834,78.151L3026.236,0l-168.273,59.423h83.692C2984.77,59.423,3021.914,65.961,3053.834,78.151z"/>
</svg>
```

**Comment by martymcguire** [comment]
This should be fixed by e417dcf02cccb24c681cc2334e172f8d4765793d

Please try grabbing the latest version and let me know if you run into this problem again!

Thanks,
Marty

**Comment by bzamecnik** [comment]
Thanks for a fast bug fix! This problem seem to be fixed. However, another exception appeared (#4). Could you please take a look? Thanks.
